From b421b6959ac541d97dc02878391d54fa274a6cb7 Mon Sep 17 00:00:00 2001
From: Artem Borisov <dedsa2002@gmail.com>
Date: Sun, 19 Aug 2018 16:15:06 +0300
Subject: [PATCH] Allow hal_usb access sysfs when in coredomain

This is necessary if we build a device-specific
USB HAL that is stored in /system/bin/hw.

Change-Id: I610cd9097ac118035a97281bb0465370a435a74d
---
 private/domain.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/private/domain.te b/private/domain.te
index fb6ba4f78..368e49cd7 100644
--- a/private/domain.te
+++ b/private/domain.te
@@ -41,6 +41,7 @@ full_treble_only(`
     -init
     -ueventd
     -vold
+    -hal_usb
   } sysfs:file no_rw_file_perms;
 
   # /dev
